docs/docs-components/PageHeader.js

```diff
@@ -132,7 +132,9 @@ export default function PageHeader({
               column: 6,
             }}
           >
-            {description && <Markdown text={description} />}
+            {description && (
+              <Markdown text={description?.replace(/https:\/\/gestalt\.pinterest\.systems/g, '')} />
```

Comment:
This replacement happens twice in this PR and once already (GeneratedPropTable) — perhaps a helper function is in order?